recipes/clfft/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "clFFT" %}
+{% set version = "2.12.2" %}
+{% set sha256 = "e7348c146ad48c6a3e6997b7702202ad3ee3b5df99edf7ef00bbacc21e897b12" %}
+
+{% set OSX_VARIANT = os.environ.get("OSX_VARIANT", "native") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/clMathLibraries/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    ############################################################
+    # Fix a type issue when using sqrt                         #
+    #                                                          #
+    # xref: https://github.com/clMathLibraries/clFFT/pull/199  #
+    ############################################################
+    - PR_199.diff
```

Comment:
This has already been accepted upstream. Was necessary to build with VC++ 2008.

xref: https://github.com/clMathLibraries/clFFT/pull/199